src/components/encoding-pane/filter-pane.scss

```diff
@@ -0,0 +1,49 @@
+$radius: 3px;
+$pill-height: 22px;
+$text-color: #212121;
+
+.filter-shelf {
+  background-color: white;
+  border-radius: $radius;
+  border: 1px solid #ddd;
+  margin-bottom: 4px;
+  max-height: 150px;
+  overflow: scroll;
+
+  :global .fa {
+    &.fa-times {
```

Comment:
Yeah, I think the `app.scss` which already the file for all `:global` stuff would be an appropriate place :)